stdlib/traceback.pyi

```diff
@@ -1,21 +1,36 @@
 import sys
 from _typeshed import SupportsWrite
 from types import FrameType, TracebackType
-from typing import IO, Any, Generator, Iterable, Iterator, List, Mapping, Optional, Tuple, Type
+from typing import IO, Any, Generator, Iterable, Iterator, List, Mapping, Optional, Tuple, Type, overload
 
 _PT = Tuple[str, int, str, Optional[str]]
 
 def print_tb(tb: TracebackType | None, limit: int | None = ..., file: IO[str] | None = ...) -> None: ...
 
 if sys.version_info >= (3, 10):
+    @overload
     def print_exception(
-        __exc: Type[BaseException] | BaseException | None,
+        __exc: Type[BaseException] | None,
         value: BaseException | None = ...,
         tb: TracebackType | None = ...,
         limit: int | None = ...,
         file: IO[str] | None = ...,
         chain: bool = ...,
     ) -> None: ...
+    @overload
+    def print_exception(
+        __exc: BaseException, *, limit: int | None = ..., file: IO[str] | None = ..., chain: bool = ...
```

Comment:
Just to verify (I didn't test it): Providing a `None` value when providing an exception object is invalid?

Reply:
It is covered by the first overload, and behaves the same on 3.9 and 3.10:

```
akuli@akuli-desktop:~$ python3.9 -c 'import traceback; traceback.print_exc(None)'
NoneType: None
akuli@akuli-desktop:~$ python3.10 -c 'import traceback; traceback.print_exc(None)'
NoneType: None
```

Reply:
Sorry for being unclear. I meant something like this: `print_exception(ValueError(), None, None, 3)`. But I just tested it on Python 3.10 and the result seems to be nonsense.